src/main/features/response/routes/eligibility/send-your-response-by-email.ts

```diff
@@ -0,0 +1,29 @@
+import * as express from 'express'
+
+import { Paths } from 'response/paths'
+
+import { ErrorHandling } from 'common/errorHandling'
+import { ResponseDraft } from 'response/draft/responseDraft'
+import { Draft } from '@hmcts/draft-store-client'
+import { FeesClient } from 'fees/feesClient'
+import { Range } from 'fees/models/range'
+
+const supportedFeeLimitInPennies: number = 1000000
+
+/* tslint:disable:no-default-export */
+export default express.Router()
+  .get(Paths.sendYourResponseByEmailPage.uri, ErrorHandling.apply(async (req: express.Request, res: express.Response) => {
+    const issueFeeGroup = await FeesClient.getIssueFeeRangeGroup()
+
+    const supportedIssueFees: Range[] = issueFeeGroup.ranges
```

Comment:
would be nice to wrap this so that we aren't repeating this logic and the supported fee limit max...
Won't REQ as this PR has been open for too long
but REC if time allows